egs/aspire/s5/local/multi_condition/aspire_prep_rir_noise_list.py

```diff
@@ -23,38 +23,48 @@ def GetArgs():
     return args
 
 
-# This function generate the rir_list file for the aspire real RIR
+# This function generates the rir_list file for the real RIRs being in ASpIRE experiments.
+# It assumes the availability of data/impulses_noises directory prepared by local/multi_condition/prepare_impulses_noises.sh
 def GenerateRirListFile(input_dir, output_dir):
   rir_list_file = open(output_dir + "/rir_list", 'w')
   rir_id = 1
-  room_id = 1
   for db in ["RVB2014", "RWCP", "air"]:
     rir_files = glob.glob(input_dir + "/{0}_*.wav".format(db))
+    rir_files.sort()
     for rir in rir_files:
       filename = rir.split('/')[-1]
       if "noise" not in filename:
-        rir_list_file.write('--rir-id {0} --room-id {1} {2}\n'.format(str(rir_id).zfill(5), str(room_id).zfill(3), rir))
+        parts = filename.split('_')
```

Comment:
Rather than parsing the file name for the necessary parameters, @tomkocse will later submit a modified list generation function which will use information available in `data/impulses_noises/info/`. This is however not a high priority change as we will preprocess the individual databases in the future and these scripts would not be part of any recipe.